src/olympia/editors/models.py

```diff
@@ -704,10 +706,46 @@ class AutoApprovalSummary(ModelBase):
     verdict = models.PositiveSmallIntegerField(
         choices=amo.AUTO_APPROVAL_VERDICT_CHOICES,
         default=amo.NOT_AUTO_APPROVED)
+    weight = models.IntegerField(default=0)
 
     def __unicode__(self):
         return u'%s %s' % (self.version.addon.name, self.version)
 
+    def calculate_weight(self):
+        """Calculate the weight value for this version according to various
+        risk factors.
+
+        That value is then used in editor tools to prioritize add-ons in the
+        auto-approved queue."""
+        # Note: for the moment, some factors are in direct contradiction with
+        # the rules determining whether or not an add-on can be auto-approved
+        # in the first place, but we'll relax those rules as we move towards
+        # post-review.
+        addon = self.version.addon
+        one_year_ago = (self.created or datetime.now()) - timedelta(days=365)
+        factors = {
+            'admin_review': 100 if addon.admin_review else 0,
+            'abuse_reports': min(
+                AbuseReport.objects
+                .filter(Q(addon=addon) | Q(user__in=addon.listed_authors))
+                .filter(created__gte=one_year_ago).count() * 10, 100),
+            'negative_reviews': min(
+                Review.objects
+                .filter(addon=addon)
+                .filter(rating__lte=3, created__gte=one_year_ago)
+                .count() * 2, 100),
```

Comment:
Yeah... I asked for clarification in the reference doc because the 1% bit wasn't clear to me, so I did this in the meantime. Doing `(.count() / 100) * 2` would result in a ridiculously low score, making that factor have very little impact on the result.